src/lang/qbittorrent_zh_HK.ts

```diff
@@ -10685,216 +10646,163 @@ Please choose a different name and try again.</source>
         <location filename="../gui/transferlistwidget.cpp" line="912"/>
         <source>&amp;Resume</source>
         <comment>Resume/start the torrent</comment>
-        <translation type="unfinished">取消暫停（&amp;R）</translation>
+        <translation>继续(&amp;R)</translation>
```

Comment:
Sorry, I can't find the right place to ask.
But I want to know How Transifex works...
Cause I see that `zh_HK` translated text has been overridden by `zh` translated text.
But it shouldn't overriding, even if it is not translated.
Since `zh_HK`/`zh_TW`/`zh`  is using different technical terms.

Reply:
https://github.com/qbittorrent/qBittorrent/wiki/How-to-translate-qBittorrent

Reply:
Sure. Translations are re-synced before every release at the very minimum.